src/styles/_header.scss

```
@@ -37,5 +37,9 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: transform 0.3s;
+    &:hover {
+      color: #00acdc;
+    }
```

здесь же нет свойства transform, зачем писать для него transition
```suggestion
    transition: color 0.3s;
    &:hover {
      color: #00acdc;
    }
```